debian/tvheadend.postinst

```diff
@@ -10,16 +10,15 @@ configure)
 
    if ! getent passwd $HTS_USER >/dev/null; then
         echo >&2 "Creating user: $HTS_USER..."
-        adduser --quiet --system --group --shell /bin/bash $HTS_USER
+        adduser --quiet --system --group --home /var/lib/tvheadend $HTS_USER
    fi
 
    HTS_HOME=`getent passwd $HTS_USER | cut -d':' -f6`
 
-   mkdir -p "${HTS_HOME}/.hts/tvheadend"
-   chown ${HTS_USER}:${HTS_USER} "${HTS_HOME}/.hts"
-   chown ${HTS_USER}:${HTS_USER} "${HTS_HOME}/.hts/tvheadend"
+   mkdir -p "${HTS_HOME}/recordings"
```

Comment:
`install` is better here

`install -d -m 0775 -g 'tvheadend' -o 'tvheadend' '/var/lib/tvheadend/recordings'` is what I'm using in my container atm

Reply:
Some limited OSs might not have the install command, so I'd leave that as is, you never know where people try to install it ;)

Reply:
Cool, I didn't know that was a thing.

Reply:
@Flole998 , "install" is a part of coreutils and has been since at least coreutils 7.  All supported Debian and Ubuntu versions have at least 8.21 and this code only ever gets executed on Debianoid systems, so I think we are good there.

Reply:
i think `install` is a posix requirement ...

Also, this will only ever be run on debian will it not, which will guaranteed have `install` :p